src/viewer/customize_the_ui.md

```diff
@@ -3,38 +3,37 @@
 ## Overview
 
 <p align="center">
-  <img width="48%" src="/assets/img/viewer/Viewer-1_window.png" alt="Viewer with one window.">
-  <img width="48%" src="/assets/img/viewer/Viewer-2_windows.png" alt="Viewer with two windows.">
+  <img width="100%" src="/assets/img/viewer/Viewer-3_windows.png" alt="Viewer with three windows.">
 </p>
 <p align="center">
-  <img width="48%" src="/assets/img/viewer/Viewer-3_windows.png" alt="Viewer with three windows.">
-  <img width="48%" src="/assets/img/viewer/Viewer-1_window_special.png" alt="Viewer with one window without header.">
+  <img width="49%" src="/assets/img/viewer/Viewer-2_windows.png" alt="Viewer with two windows.">
+  <img width="49%" src="/assets/img/viewer/Viewer-1_window_special.png" alt="Viewer with one window without header.">
 </p>
 
 The viewer UI is composed of three main parts :
 
-- First, there is the [header](#header). The header is where some useful actions can be performed like accessing the main menu, loading models or customizing the rest of the UI using the window manager. The header is not displayed on the bottom-right image. It is a special case where user only wants a floating header and a button (top-right) to customize windows.
+- First, there is the [menu](#menu). The menu is where some useful actions can be performed like customizing the rest of the UI using the window manager. The menu is not displayed on the bottom-right image. It is a special case where the user wants to hide it. By default, a button (top-right) is automatically added to customize the windows.
 - Next, there is the main view where [windows](#window) are displayed. The user can resize, add, delete, split or swap windows to create his own workspace, according to his needs. On the images above there is one, two or three windows with spatial arborescence, a 2d and 3d representations of the model displayed on different windows.
 - Last, there are [plugins](#plugin) displayed on windows. Different display possibilities are shown on the image below (full, left, right, small...).
 
 <p align="center">
   <img width="70%" src="/assets/img/viewer/Viewer-1_window_plugins.png" alt="Viewer with opened plugins.">
 </p>
 
-## Header
+## Menu
 
-By default, the header is always displayed as a top bar. However, it can be changed if only one window is present, or totally removed if you need to.
+By default, the emnu is always displayed as a top bar. However, it can be changed if only one window is present, or totally removed if you need to.
```

Comment:
```suggestion
By default, the menu is always displayed as a top bar. However, it can be changed if only one window is present, or totally removed if you need to.
```